core/controllers/editor.py

```diff
@@ -312,10 +325,11 @@ def get(self, exploration_id):
         if not rights_manager.Actor(self.user_id).can_view(
                 rights_manager.ACTIVITY_TYPE_EXPLORATION, exploration_id):
             raise self.PageNotFoundException
-
         version = self.request.get('v', default_value=None)
+        apply_draft = self.request.get('apply_draft', default_value=False)
```

Comment:
@jacobdavis11 Apparently the history tab calls this same handler to get() previous versions of the exploration (this happens when there is a 'v' argument). Should we make a separate handler for the tab, or is an option fine here?

@sbhowmik89 in any case, this could do with more documentation about context. I think it's really important to be clear here.